add MFA detail (#884)
astronomer/docs: astro/configure-idp.md

- Users will automatically be authenticated to Astro if they're already logged in to your IdP.
- Users will no longer have to repeatedly login and remember credentials for their account.
- You will have complete ownership over credential configuration and management on Astro.

> - You have the option of enforcing Multi-factor authentication (MFA) for your users.

Reviewer: ```suggestion
- You can enforce multi-factor authentication (MFA) for users.
```
